segment_images.py

```diff
@@ -100,7 +112,9 @@ def main(args):
 
     img_folder = Path(args.data_path, args.dataset, args.split)
 
-    output_name = f"{args.dataset}_{args.split}_{args.size}"
+    output_name = (
+        f"{args.dataset}_{args.split}_{args.size}_{args.points_per_side}_{args.thresh}"
```

Comment:
Might want to add parameter names
```suggestion
        f"{args.dataset}_{args.split}_{args.size}_points-per-side_{args.points_per_side}_min-area_{args.thresh}"
```